README.md

```diff
@@ -1,2 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
 # OOP-school-library
-This app allows you to:  Add new students or teachers. Add new books. Save records of who borrowed a given book and when. This project was part of my Microverse journey and it was designed to make me practice Object Oriented Programming while building a app based on a UML class diagram..
+
+This app allows you to: 
+- Add new students or teachers. 
+- Add new books. 
+- Save records of who borrowed a given book and when. 
+
+This project was part of my Microverse journey and it was designed to make me practice Object Oriented Programming while building a app based on a UML class diagram with Ruby.
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+To run this project follow these simple steps:
+
+- Clone this repo with git, or [download the zip folder](https://github.com/tiagomarin/OOP-school-library/archive/refs/heads/add-classes-Person-Student-Teacher.zip) and extract to your local folders.
+- Run the app.rb file on the terminal and follow the step by step answering the questions.
+
+
+### Prerequisites
+
+- Ruby installed in your computer. ([check this guide if you need to install](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md))
+- Any code editor like Vscode or Vim.
+
+
+## Authors
```

Comment:
- [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.

Reply:
Thanks for pointing that out! I'll apply this change also!